### PR TITLE
sql-stmt: update synopsis to ebnf diagrams for 6 statements

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Update synopsis to ebnf diagrams for 6 SQL statements, to make it clearer, and easier to maintain 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
  - https://github.com/pingcap/docs/pull/17085
  - https://github.com/pingcap/docs/pull/17089
  - https://github.com/pingcap/docs/pull/17093
  - https://github.com/pingcap/docs/pull/17095
  - https://github.com/pingcap/docs/pull/17097
  - https://github.com/pingcap/docs/pull/17099
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-show-indexes.md [hunk 1]
```diff
@@ -10,25 +10,14 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-indexes/','/docs-cn/de
 
 ## 语法图
 
-**ShowIndexStmt:**
+```ebnf+diagram
+ShowIndexStmt ::=
+    "SHOW" ( "INDEX" | "INDEXES" | "KEYS" ) ("FROM" | "IN" ) TableName (("FROM" | "IN") SchemaName )? ShowLikeOrWhereOpt?
```
Comment (kennytm) [inline sql-statements/sql-statement-show-indexes.md:15]:
```suggestion
    "SHOW" ( "INDEX" | "INDEXES" | "KEYS" ) ("FROM" | "IN" ) TableName (("FROM" | "IN") SchemaName )? ShowLikeOrWhere?
```

Comment (lilin90) [reply sql-statements/sql-statement-show-indexes.md]: I'll create another PR later to update this place in English document, which is ignored in https://github.com/pingcap/docs/pull/17085/files.

**Review by kennytm — APPROVED** [review]

File: sql-statements/sql-statement-split-region.md
Comment (qiancai) [inline sql-statements/sql-statement-split-region.md:1]: 英文 PR https://github.com/pingcap/docs/pull/17097/files 里删除了这篇文档里的  {{< copyable "sql" >}}，中文也可以对应删除

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**